autoPyTorch/pipeline/components/training/data_loader/feature_data_loader.py

```diff
@@ -93,5 +93,4 @@ def _check_transform_requirements(self, X: Dict[str, Any], y: Any = None) -> Non
                 mechanism, in which during a transform, a components adds relevant information
                 so that further stages can be properly fitted
         """
-        if not X['dataset_properties']['is_small_preprocess'] and 'preprocess_transforms' not in X:
```

Comment:
Can we continue this test by just removing `not X['dataset_properties']['is_small_preprocess']`?